### PR TITLE
pig-latin: tests for Y as a vowel

### LINKED ISSUE #900: pig-latin tests: 'y' as first vowel in a word (and not first letter).
Ther are two rules for letter 'y':
1. when 'y' is first letter of a word (eg. "yellow") - test case exists.
2. when 'y' is first vowel but not first letter of a word (eg. "tyrell") - test case doesn't exists.

### PR DESCRIPTION
**Pull request by pnie**
Fixes #900.

### CODE REVIEW

File: exercises/pig-latin/canonical-data.json [hunk 1]
```diff
@@ -137,6 +131,29 @@
...
+      "description": "position of y in a word determines if it is a consonant or a vowel",
+      "cases": [
+        {
+          "description": "y at beginning of a word is treated like a consonant",
+          "property": "translate",
+          "input": "yellow",
+          "expected": "ellowyay"
+        },
+        {
+          "description": "at end of a consonant cluster, y is treated like a vowel",
```
Comment (rpottsoh) [inline exercises/pig-latin/canonical-data.json]: > "at end of a consonant cluster, y is treated like a vowel"

Turn this around.
```
"y is treated like a vowel at the end of a consonant cluster"
```

File: exercises/pig-latin/canonical-data.json [hunk 2]
```diff
@@ -137,6 +131,29 @@
       ]
     },
     {
+      "description": "position of y in a word determines if it is a consonant or a vowel",
+      "cases": [
+        {
+          "description": "y at beginning of a word is treated like a consonant",
```
Comment (rpottsoh) [inline exercises/pig-latin/canonical-data.json]:
```
"y is treated like a consonant at the beginning of a word "
```

Comment (Vankog) [reply exercises/pig-latin/canonical-data.json]: @rpottsoh Could you please elaborate, why this is necessary? (same for the other comment)
I am a bit confused about why this matters and what syntax we should aim for. Is there a guideline somewhere?

**Comment by petertseng** [comment]
don't forget to change the version as https://github.com/exercism/problem-specifications/blob/master/README.md#test-data-versioning

**Comment by rpottsoh** [comment]
@pnie thanks for working on this.  Good work.

File: exercises/pig-latin/canonical-data.json (hunk 2 above)
Comment (rpottsoh) [reply exercises/pig-latin/canonical-data.json]: @Vankog the sentence was lacking a comma.  By reordering the wording it simplifies the statement and removes the need for a comma.  Also, several tracks employ test generators.  Often times `"description"` becomes the method name of the test.

Comment (Vankog) [reply exercises/pig-latin/canonical-data.json]: I see, thanks

**Pull request merged**